sci-biology/squizz/squizz-0.99b.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+DESCRIPTION="squizz is a sequence format file checker, but it has some
+conversion capabilities too."
+HOMEPAGE=""
+SRC_URI="ftp://ftp.pasteur.fr/pub/gensoft/projects/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+src_configure(){
+    econf
+}
+
+src_compile() {
+    emake 
+}
```

Comment:
Default since EAPI=2, just drop this phase.
https://devmanual.gentoo.org/ebuild-writing/functions/src_compile/index.html